main/modes/games/bigbug/soundManager_bigbug.h

```diff
@@ -0,0 +1,59 @@
+#ifndef _SOUNDMANAGER_BIGBUG_H_
+#define _SOUNDMANAGER_BIGBUG_H_
+
+//==============================================================================
+// Includes
+//==============================================================================
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <spiffs_song.h>
```

Comment:
THis no longer exists and sounds wisll need to be MIDI-ified. @dylwhich can guide that

Reply:
Yeah, for MIDI there's a bit in the [porting docs](https://adam.feinste.in/Super-2024-Swadge-FW/porting.html#things-that-need-to-be-intelligently-changed-or-replaced) but it should just be:
* Replace `song_t` with `midiFile_t`
* Replace `loadSong()` with `loadMidiFile()`
* Replace `freeSong()` with `unloadMidiFile()`
* You can continue using `soundPlayBgm()` and `soundPlaySfx()` - they just take `midiFile_t*` instead of `song_t*`

Reply:
> fatal error: spiffs_song.h: No such file or directory

I also tried fs_song.h: No such file. What should I include instead?

Reply:
You'd need to `git merge --squash origin/main` to get that file. Let me see if I can do that for ya.

Reply:
Oh, actually it's `midiFileParser.h`. I gotchu

Reply:
OK, we're back on track. CI is doing it's thing. I also fixed some memory leaks and warnings, just because.